lib/disposable.dart

```diff
@@ -12,5 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export 'package:w_common/src/disposable/disposable.dart';
-export 'package:w_common/src/disposable/disposable_manager.dart';
+/// This entry point has been split into separate VM and browser entry points;
+/// use disposable_vm.dart or disposable_browser.dart instead.
+@deprecated
```

Comment:
I'm restructuring in this next commit so no deprecation will occur. (We actually weren't really sure if you can deprecate like this anyway)